### PR TITLE
[DOCS-6271] ADW 2.2

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by DenysVuika — DISMISSED** [review]

File: digital-workspace/2.1/process/managing.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Managing Processes
```
Comment (mauriziovitale) [inline digital-workspace/2.1/process/managing.md:2]: shouldn't we have a note that says by default the PROCESS_SERVICE_PLUGIN is disabled and you need to turn it on. Of course as pre-requisite, you must have an activiti instance up and running on the env.

Comment (oliveralfresco2018) [reply digital-workspace/2.1/process/managing.md:2]: @mauriziovitale 

Thanks, that property is mentioned on the config.md file. 
Although it looks like the name I was given when initially writing the docs was different.
Can you confirm the correct name please? In the current docs the name is processService ?
Cheers. 

Note: Did the name change between versions also ? i.e. 2.1 may require a diff name to 2.2 ?

**Review by anxumalo — APPROVED** [review]

File: digital-workspace/2.1/process/managing.md (hunk 1 above)
Comment (mauriziovitale) [reply digital-workspace/2.1/process/managing.md:2]: "plugins": {
    "microsoftOnline": false,
    "processService": "true"
  },

Comment (mauriziovitale) [reply digital-workspace/2.1/process/managing.md:2]: Sorry @oliveralfresco2018  I was wrong the name didn't changed

**Pull request merged**